### PR TITLE
Simplify rating loading

### PR DESCRIPTION
**Pull request by Bond-009**
This doesn't save the ratings to disk anymore and load them directly from the resources.

### CODE REVIEW

File: Emby.Server.Implementations/Localization/LocalizationManager.cs [hunk 1]
```diff
@@ -74,100 +70,42 @@ public async Task LoadAll()
...
+                        if (string.IsNullOrWhiteSpace(line))
+                        {
+                            continue;
+                        }
+
+                        string[] parts = line.Split(',');
+                        if (parts.Length == 2
+                            && int.TryParse(parts[1], NumberStyles.Integer, UsCulture, out var value))
+                        {
+                            dict.Add(parts[0], (new ParentalRating { Name = parts[0], Value = value }));
```
Comment (cvium) [inline Emby.Server.Implementations/Localization/LocalizationManager.cs]: Is this parenthesis necessary?

Comment (Bond-009) [reply Emby.Server.Implementations/Localization/LocalizationManager.cs]: Nope

File: Emby.Server.Implementations/Localization/LocalizationManager.cs [hunk 2]
```diff
@@ -74,100 +70,42 @@ public async Task LoadAll()
...
+                        string[] parts = line.Split(',');
+                        if (parts.Length == 2
+                            && int.TryParse(parts[1], NumberStyles.Integer, UsCulture, out var value))
+                        {
+                            dict.Add(parts[0], new ParentalRating { Name = parts[0], Value = value });
+                        }
+#if DEBUG
+                        else
+                        {
+                            _logger.LogWarning("Misformed line in ratings file for country {CountryCode}", countryCode);
```
Comment (JustAMan) [inline Emby.Server.Implementations/Localization/LocalizationManager.cs]:
```suggestion
                            _logger.LogWarning("Malformed line in ratings file for country {CountryCode}", countryCode);
```

File: Emby.Server.Implementations/Localization/LocalizationManager.cs [hunk 3]
```diff
@@ -74,100 +70,42 @@ public async Task LoadAll()
...
+                        }
+#endif
                     }
                 }
-            }
+                _logger.LogWarning("{t}", countryCode);
```
Comment (JustAMan) [inline Emby.Server.Implementations/Localization/LocalizationManager.cs]: eh?..

Comment (Bond-009) [reply Emby.Server.Implementations/Localization/LocalizationManager.cs]: Whoops, log line for testing

**Pull request merged**